core/invoker/Dockerfile

```diff
@@ -1,8 +1,10 @@
 FROM scala
 
 # Uncomment to fetch latest version of docker instead: RUN wget -qO- https://get.docker.com | sh
-# Install docker 1.9.0
-RUN wget --no-verbose -O /usr/bin/docker "https://get.docker.com/builds/Linux/x86_64/docker-1.9.0" && \
+# Install docker 1.12.0 Client
+RUN wget --no-verbose https://get.docker.com/builds/Linux/x86_64/docker-1.12.0.tgz && \
+tar --strip-components 1 -xvzf docker-1.12.0.tgz -C /usr/bin docker/docker && \
+rm -f docker-1.12.0.tgz && \
```

Comment:
Is there a chance that these stay installable like that? So we could have an ENV variable in here

`DOCKER_VERSION=1.12.0` and use it throughout the URLs for an easier upgrade in the future? What do you think?